samples/advanced-csharp/Controllers/ApiController.cs

```diff
@@ -31,6 +32,24 @@ public ApiController(Microsoft.Extensions.Configuration.IConfiguration configura
         [Route("token")]
```

Comment:
Need to add [HttpPost] here?

Reply:
Added, as well as content types supported.